### PR TITLE
Generate bb bw and vcf files when there are no reads in the bam file

### PR DESCRIPTION
**Pull request by cadley-nyulangone**

### CODE REVIEW

File: dnase/analysis.sh [hunk 1]
```diff
@@ -439,7 +439,6 @@ elif [[ "${sampleType}" == "dnase" ]] || [[ "${sampleType}" == "atac" ]] || [[ "
         fi
         #Kent tools can't use STDIN
         bedGraphToBigWig $TMPDIR/${name}.${mappedgenome}.perBase.clipped.bedGraph ${chromsizes} ${sampleOutdir}/${name}.${mappedgenome}.perBase.bw
-
```
Comment (mattmaurano) [inline dnase/analysis.sh:442]: stray line

File: dnase/callsnpsMerge.sh [hunk 2]
```diff
@@ -72,10 +72,15 @@ date
 #NB: for hg38_full, many jobs will not generate coverage tracks
 covfiles=`cut -f1 ${sampleOutdir}/inputs.callsnps.${mappedgenome}.txt | xargs -I {} echo "${sampleOutdir}/${name}.${mappedgenome}.{}.coverage.starch"`
 covfiles=$(getFilesToMerge ${covfiles})
-starchcat ${covfiles} > ${sampleOutdir}/${name}.${mappedgenome}.coverage.starch
-rm -f ${covfiles}
 
-unstarch ${sampleOutdir}/${name}.${mappedgenome}.coverage.starch | cut -f1-3,5 > $TMPDIR/${name}.${mappedgenome}.coverage.bedGraph
+if [ "${covfiles}" = "" ]; then
```
Comment (mattmaurano) [inline dnase/callsnpsMerge.sh:76]: why would this be empty? doesn't your change above ensure inputs.callsnps has entries?

**Review by mattmaurano — CHANGES_REQUESTED** [review]

File: dnase/analysis.sh (hunk 1 above)
Comment (cadley-nyulangone) [reply dnase/analysis.sh:442]: Not sure why line 442 looks that way on the right, in green.  There are only 2 tabs in the code.

**Comment by cadley-nyulangone** [comment]
Yes, you're right.  I pulled out all those if blocks from callsnpsMerge.sh
  Bad weekend.

File: dnase/analysis.sh (hunk 1 above)
Comment (mattmaurano) [reply dnase/analysis.sh:442]: it looks like you added whitespace -- you're saying that's not the case?

**Comment by mattmaurano** [comment]
No worries. Can you double check by running a good and bad sample from the latest FC?

**Comment by cadley-nyulangone** [comment]
I had added tabs, but it wanted whitespace.  Fixed now.

**Comment by cadley-nyulangone** [comment]
OK

**Comment by cadley-nyulangone** [comment]
Ran *BS08612A.cegsvectors_LP300* (good) and
*BS08612A.cegsvectors_pSpCas9GFP* (bad).  Both working now.

**Pull request merged**